File: Assignment 4/Q1.c
#include <stdio.h>

int main(){
    int arr1[100], n, i;
    printf("Enter size of array: ");
    scanf("%d", &n);
    printf("Enter elements: ");
    for (i = 0; i < n; i++) 
    scanf("%d", &arr1[i]);
    printf("Array elements: ");
    for (i = 0; i < n; i++) 
    printf("%d ", arr1[i]);
    return 0;
}

File: Assignment 4/Q1.py
arr1 = []
n = int(input("Enter size of array: "))
print("Enter elements: ")
for i in range(n):
    element = int(input())
    arr1.append(element)
print("Array elements: ", end="")
for i in range(n):
    print(arr1[i], end=" ")
